packages/forklift-console-plugin/src/modules/Providers/views/details/components/InventorySection/VSphereInventorySection.tsx

```diff
@@ -16,6 +16,10 @@ export const VSphereInventorySection: React.FC<InventoryProps> = ({ data }) => {
   }
 
   const inventoryItems = {
+    product: {
```

Comment:
keep the placement of items similar to other providers

Reply:
`product` can either go first or last - should I put at the end?

Reply:
hmm, ok, first